src/views/CourtOrder.vue

```diff
@@ -485,8 +489,7 @@ export default class CourtOrderView extends Mixins(DateMixin, FilingMixin, Commo
   /** Called when component is created. */
   created (): void {
     // Safety check to make sure Staff is filing the Continuation Out.
-    if (!this.isRoleStaff) {
-      this.resumeErrorDialog = true
+    if (!this.IsAuthorized(AuthorizedActions.CONTINUATION_OUT_FILING)) {
```

Comment:
Wrong filing.

Reply:
Still the wrong filing but I added another comment for that so I'm resolving this one.